Tools/Scripts/compare-static-analysis-results

```diff
@@ -94,19 +100,18 @@ def compare_project_results(args, archive_path, new_path, project):
         new_issues_total.update(new_issues)
         new_files_total.update(new_files)
 
-        print(f'    Fixed {len(fixed_issues)} issue(s).')
-        print(f'    Fixed {len(fixed_files)} file(s).')
-        print(f'    {len(new_issues)} new issue(s).')
-        print(f'    {len(new_files)} new file(s) with issues.\n')
+        print(f'    Issues fixed: {len(fixed_issues)}')
+        print(f'    Files fixed: {len(fixed_files)}')
+        print(f'    New issues: {len(new_issues)}')
+        print(f'    New dirty files: {len(new_files)}')
```

Comment:
I don't think we wanna use this terminology in developer facing UI. Probably says "files with new issues" or something.